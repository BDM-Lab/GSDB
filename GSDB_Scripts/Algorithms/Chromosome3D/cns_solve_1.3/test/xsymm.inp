remarks:  file xsymm.inp -- tests symmetry operator parser


xray
  symmetry=( X,Y,Z )
  symmetry=( -X,-Y,1/2+Z )
  symmetry=( -X,Y,1/2-Z )
  symmetry=( X,-Y,-Z )
  symmetry=( 1/2+X,1/2+Y,Z )
  symmetry=( 1/2-X,1/2-Y,1/2+Z )
  symmetry=( 1/2-X,1/2+Y,1/2-Z )
  symmetry=( 1/2+X,1/2-Y,-Z )

  symmetry ?
  ?
  display  $symmetry  

  display  $symmetry_op_1
  display  $symmetry_op_2
  display  $symmetry_op_3
  display  $symmetry_op_4
  display  $symmetry_op_5
  display  $symmetry_op_6
  display  $symmetry_op_7
  display  $symmetry_op_8




  symmetry reset

  symmetry=( X,Y,Z )
  symmetry=( -X,-Y,Z )
  symmetry=( 1/2-X,1/2+Y,-Z )
  symmetry=( 1/2+X,1/2-Y,-Z )

  symmetry ?
  display  $symmetry  


end

stop


