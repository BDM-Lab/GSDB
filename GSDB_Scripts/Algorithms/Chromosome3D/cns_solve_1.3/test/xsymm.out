          ============================================================
          |                                                          |
          |            Crystallography & NMR System (CNS)            |
          |                         CNSsolve                         |
          |                                                          |
          ============================================================
           Version: 1.3
           Status: General release
          ============================================================
           Written by: A.T.Brunger, P.D.Adams, G.M.Clore, W.L.DeLano,
                       P.Gros, R.W.Grosse-Kunstleve,J.-S.Jiang,J.M.Krahn,
                       J.Kuszewski, M.Nilges, N.S.Pannu, R.J.Read,
                       L.M.Rice, G.F.Schroeder, T.Simonson, G.L.Warren.
           Copyright (c) 1997-2010 Yale University
          ============================================================
           Running on machine: lewis4-r710-login-node223 (x86_64/Linux,64-bit)
                               with 16 threads
           Program started by: oeow39
           Program started at: 16:08:42 on 05-Jul-2018
          ============================================================

 FFT3C: Using FFTPACK4.1

 CNSsolve>remarks:  file xsymm.inp -- tests symmetry operator parser 
 CNSsolve> 
 CNSsolve> 
 CNSsolve>xray 
 XRAY>  symmetry=( X,Y,Z ) 
 XRAY>  symmetry=( -X,-Y,1/2+Z ) 
 XRAY>  symmetry=( -X,Y,1/2-Z ) 
 XRAY>  symmetry=( X,-Y,-Z ) 
 XRAY>  symmetry=( 1/2+X,1/2+Y,Z ) 
 XRAY>  symmetry=( 1/2-X,1/2-Y,1/2+Z ) 
 XRAY>  symmetry=( 1/2-X,1/2+Y,1/2-Z ) 
 XRAY>  symmetry=( 1/2+X,1/2-Y,-Z ) 
 XRAY> 
 XRAY>  symmetry ? 
 | Symmetry operators:
 | -------------------
 | SYMMetry=(X,Y,Z)
 | SYMMetry=(-X,-Y,Z+1/2)
 | SYMMetry=(-X,Y,-Z+1/2)
 | SYMMetry=(X,-Y,-Z)
 | SYMMetry=(X+1/2,Y+1/2,Z)
 | SYMMetry=(-X+1/2,-Y+1/2,Z+1/2)
 | SYMMetry=(-X+1/2,Y+1/2,-Z+1/2)
 | SYMMetry=(X+1/2,-Y+1/2,-Z)
 XRAY>  ? 
 ---------------------------diffraction-data-----------------------------------
 | a=  1.00, b=  1.00, c=  1.00, alpha= 90.00, beta= 90.00, gamma= 90.00
 | ANOMalous=FALSe
 | Number or unique reflections read=     0
 | resolution range covered by these reflections:       0.00 to       0.00
 | Hmax=    0 Kmax=    0 Lmax=    0 Hmin=    0 Kmin=    0 Lmin=    0
 ------------------------------------------------------------------------------
 XRAY>  display  $symmetry 
 8
 XRAY> 
 XRAY>  display  $symmetry_op_1 
 (X,Y,Z)
 XRAY>  display  $symmetry_op_2 
 (-X,-Y,1/2+Z)
 XRAY>  display  $symmetry_op_3 
 (-X,Y,1/2-Z)
 XRAY>  display  $symmetry_op_4 
 (X,-Y,-Z)
 XRAY>  display  $symmetry_op_5 
 (1/2+X,1/2+Y,Z)
 XRAY>  display  $symmetry_op_6 
 (1/2-X,1/2-Y,1/2+Z)
 XRAY>  display  $symmetry_op_7 
 (1/2-X,1/2+Y,1/2-Z)
 XRAY>  display  $symmetry_op_8 
 (1/2+X,1/2-Y,-Z)
 XRAY> 
 XRAY> 
 XRAY> 
 XRAY> 
 XRAY>  symmetry reset 
 XRAY> 
 XRAY>  symmetry=( X,Y,Z ) 
 XRAY>  symmetry=( -X,-Y,Z ) 
 XRAY>  symmetry=( 1/2-X,1/2+Y,-Z ) 
 XRAY>  symmetry=( 1/2+X,1/2-Y,-Z ) 
 XRAY> 
 XRAY>  symmetry ? 
 | Symmetry operators:
 | -------------------
 | SYMMetry=(X,Y,Z)
 | SYMMetry=(-X,-Y,Z)
 | SYMMetry=(-X+1/2,Y+1/2,-Z)
 | SYMMetry=(X+1/2,-Y+1/2,-Z)
 XRAY>  display  $symmetry 
 4
 XRAY> 
 XRAY> 
 XRAY>end 
 CNSsolve> 
 CNSsolve>stop 
          ============================================================
           Maximum dynamic memory allocation:     8201488 bytes
           Maximum dynamic memory overhead:           256 bytes
           Program started at: 16:08:42 on 05-Jul-2018
           Program stopped at: 16:08:43 on 05-Jul-2018
           CPU time used:       0.0043 seconds
          ============================================================
